### PR TITLE
chore(docs): card page new format (VIV-2086)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Review by TaylorJ76 — COMMENTED** [review]
Make sure the headings are in [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)

[Online tool](https://capitalizemytitle.com/style/APA/)

File: libs/components/src/lib/card/README.md [hunk 1]
```diff
@@ -217,91 +159,43 @@ The card subtitle can be trimmed to your preferable number of lines.
...
-</vwc-layout>
-```
+### Properties
+
+<div class="table-wrapper">
+
+| Name           | Type                                      | Description                   |
+| -------------- | ----------------------------------------- | ----------------------------- |
+| **appearance** | `elevated` (default), `outlined`, `ghost` | Sets the element's appearance |
+| **elevation**  | `2`, `4` (default), `8`, `12`, `16`, `24` | Sets the element's elevation  |
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/README.md:185]: What about `headline`, `subtitle` and `text`?

File: libs/components/src/lib/card/README.md [hunk 2]
```diff
@@ -144,13 +81,13 @@ The media slot can be used to display images or video content above the card hea
 </vwc-card>
 ```
 
-### Meta
+### Meta Slot
 
 The meta slot is for action content in the card header.
 
 ```html preview
 <vwc-card
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/README.md:89]: Maybe take this example a bit further and add a menu to the slotted button?

File: libs/components/src/lib/card/USE-CASES.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+	}
+	.learn-more {
+		margin-block-start: 24px;
+	}
+</style>
+```
+
+### Selectable cards
+
+You can slot a card with `appearance="ghost""` inside selectable box
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/USE-CASES.md:153]:
```suggestion
You can slot a card with `appearance="ghost"` inside selectable box
```

File: libs/components/src/lib/card/VARIATIONS.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+## Card Texts
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/VARIATIONS.md:1]:
```suggestion
## Card Text Content
```

File: libs/components/src/lib/card/VARIATIONS.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+<vwc-card
+	text="The card can contain multiple lines of text."
+	headline="Vivid Card Component"
+	subtitle="Extra text below the card headline"
+></vwc-card>
+```
+
+## Icon
+
+The `icon` attribute displays an icon from the [icon library](/icons/icons-gallery), which prefixes the Text Field's input element.
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/VARIATIONS.md:36]: The text references the Text Field... easy mistake to make :)

File: libs/components/src/lib/card/README.md [hunk 6]
```diff
@@ -125,13 +62,13 @@ The graphic slot overrides the icon property.
...
+### Media Slot
 
 The media slot can be used to display images or video content above the card header.
 
 ```html preview
 <vwc-card
-	headline="Card with Media"
+	headline="Card with Media Slot"
 	subtitle="Extra text below the card headline"
 	style="max-inline-size: 300px"
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/README.md:73]: Can we use a class here?

File: libs/components/src/lib/card/USE-CASES.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
+## Card UI
```
Comment (TaylorJ76) [inline libs/components/src/lib/card/USE-CASES.md:1]: I don't think we need this heading, just make both the heading H2s

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/README.md (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/card/README.md:185]: I did the same as button and text-field - where the label and other text prop not in the list 
https://vivid.deno.dev/components/button/code/#properties
https://vivid.deno.dev/components/text-field/code/#properties

Comment (TaylorJ76) [reply libs/components/src/lib/card/README.md:198]: I'll go back over those then. Thanks for pointing that out.

File: libs/components/src/lib/card/GUIDELINES.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
+## Advantages of card UI design
```
Comment (rachelbt) [inline libs/components/src/lib/card/GUIDELINES.md:1]:
```suggestion
## Advantages of Card UI Design
```

File: libs/components/src/lib/card/GUIDELINES.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+Advantages of card UI design
+
+### Highly responsive
```
Comment (rachelbt) [inline libs/components/src/lib/card/GUIDELINES.md:5]:
```suggestion
### Highly Responsive
```

File: libs/components/src/lib/card/GUIDELINES.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+Probably the main advantage of UI cards is that they are very adaptive. It’s the rectangular shape that makes them transformative. That is why UI cards look good on all screen sizes.
+
+### Easy-to-understand
```
Comment (rachelbt) [inline libs/components/src/lib/card/GUIDELINES.md:9]:
```suggestion
### Easy-To-Understand
```

File: libs/components/src/lib/card/GUIDELINES.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+
+UI cards are rather minimalistic and understandable design elements. Small chunks of information they reveal are easy to consume.
+
+### Intuitive and UX friendly
```
Comment (rachelbt) [inline libs/components/src/lib/card/GUIDELINES.md:13]:
```suggestion
### Intuitive and UX Friendly
```

File: libs/components/src/lib/card/GUIDELINES.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+
+Most users are familiar with the card interface design and can easily navigate through it. Rectangular shape and simple clickable elements make the user experience very intuitive.
+
+### Attractive and modern-looking
```
Comment (rachelbt) [inline libs/components/src/lib/card/GUIDELINES.md:17]:
```suggestion
### Attractive and Modern-Looking
```

File: libs/components/src/lib/card/USE-CASES.md [hunk 13]
```diff
@@ -0,0 +1,206 @@
+## Cards inside a layout
```
Comment (rachelbt) [inline libs/components/src/lib/card/USE-CASES.md:1]:
```suggestion
## Cards Inside a Layout
```

File: libs/components/src/lib/card/USE-CASES.md [hunk 14]
```diff
@@ -0,0 +1,206 @@
...
+	.card-badge {
+		margin-block-start: 0;
+	}
+	.learn-more {
+		margin-block-start: 24px;
+	}
+</style>
+```
+
+## Selectable cards
```
Comment (rachelbt) [inline libs/components/src/lib/card/USE-CASES.md:149]:
```suggestion
## Selectable Cards
```

File: libs/components/src/lib/card/README.md (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/card/README.md:198]: added

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**